### PR TITLE
Adding libconeangle

### PR DESCRIPTION
**Pull request by kjelljorner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x]  Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kjelljorner** [comment]
@conda-forge-admin, please ping team

File: recipes/libconeangle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/libconeangle-{{ version }}.tar.gz  # [unix]
+  sha256: bc828be92fdf2d2d353b5e8bb95644068220d92809276312ff2d7bca0aa8b2d1  # [unix]
+  url: https://pypi.org/packages/cp{{ CONDA_PY }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-cp{{ CONDA_PY }}-cp{{ CONDA_PY }}-win_amd64.whl  # [win]
```
[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: The problem is conda-forge's GFortran compiler version 5.3 on Windows, which is not sufficient to build the [Fortran stdlib](https://github.com/fortran-lang/stdlib) dependency. I'm therefore using the wheels that I [build](https://github.com/kjelljorner/libconeangle/actions/workflows/build_wheels.yml) on GitHub Actions runners with GFortran 11.2 (with static linking).

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/libconeangle/meta.yaml:11]: Do you mind waiting for an input from out Fortran expert (@isuruf). Maybe we have an alternative that may work.

[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: Sure! Of course the final solution would be to have a modern Fortran compiler for Windows 😉

[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: @ocefpaf Of course I do not mind waiting for a reasonable amount of time, but we have no response from @isuruf in two weeks. Unless you have a Fortran compiler for Windows (newer than GFortran 5.3) I don't really see any option to build this on conda-forge.

[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: I tried to install it into a conda environment with on my Windows machine, and it didn't work at all. But there seems to be some [problem](https://github.com/mamba-org/mamba/issues/897) with mamba and flang. Maybe I can give it another try.

[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: I tried it now again, starting a fresh new conda env and installing `flang`, `cmake` and `ninja`. I can't even get CMake to recognize `flang`, even with `set FC=flang`. Do you have any instructions for how to use it? Can maybe copy in @awvwgk who might have some experience.

[the remark this answers is not in the document]
Comment (awvwgk) [reply recipes/libconeangle/meta.yaml:11]: The classic flang compiler is unlikely to work here as the library uses features which the PGFortran frontend is unlikely to parse at all (Fortran 2008 and newer standard). Not sure whether conda-forge already has the newer LLVM flang available, while for this one the frontend is pretty complete, last time I tried (LLVM flang 13 via MSYS2) the codegen was still missing.

[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: @isuruf Considering that the classic flang cannot compile stdlib, and that the new flang is still far from a fully functional release, could we proceed with the wheels solution for now?

[the remark this answers is not in the document]
Comment (isuruf) [reply recipes/libconeangle/meta.yaml:11]: Yeah, wheels are okay for now.

[the remark this answers is not in the document]
Comment (kjelljorner) [reply recipes/libconeangle/meta.yaml:11]: @ocefpaf Considering the input from @isuruf, would you be OK with proceeding with this?

**Pull request merged**